### PR TITLE
Auto-generate some users for load testing

### PR DESCRIPTION
**Pull request by kumar303**
Supports https://github.com/mozilla/olympia/issues/1153

### CODE REVIEW

**Comment by kumar303** [comment]
@mstriemer When I'm load testing, it will be less convenient to deal with FxA logins. Could I set `fxa_id` on these user accounts (or something) to force them to do the old style of login?

**Comment by mstriemer** [comment]
If you set `fxa_id` then the user will always use FxA for login. I assume you're testing against dev?

If you aren't testing login you could borrow the secret key and create the session without actually logging in. FxA has a pre-verified API which we could use to create the user's account without confirming the email address if that helps.

File: apps/zadmin/management/commands/gen_loadtest_users.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                'User file exists: {}. Delete or move it first.'.format(fn))
+        group_id = os.urandom(3).encode('hex')
+
+        with open(fn, 'w') as user_file:
+            print 'About to generate {} users; prefix={}'.format(
+                options['total'], group_id)
+            for index in range(options['total']):
+                user_id = '{}-{}'.format(group_id, index)
+                username = 'loadtest-{}'.format(user_id)
+                email = '{}@nowhere.org'.format(username)
```
Comment (mstriemer) [inline apps/zadmin/management/commands/gen_loadtest_users.py]: `nowhere.org` looks like a valid domain. Should we use something like `addons.mozilla.org`?

**Comment by mstriemer** [comment]
Looks good to me, r+

You could create the session while you're at it I'd imagine. Create a session cookie and drop that in the file too.

**Comment by kumar303** [comment]
> Create a session cookie and drop that in the file too.

Huh! Interesting idea. Cookie sessions for the win.

**Pull request merged**